scripts/animation_test.jl

```diff
@@ -1,11 +1,13 @@
 include("temperature_render.jl")
 
-angles = 75.0:1.0:85.0
-# angles = [-179.0:5.0:-1.0;1.0:5.0:179.0]
-n_frames = length(angles)
-anim = @animate for (i, angle) in enumerate(angles)
+# angles = 75.0:1.0:85.0
+# angles = [-179.0:1.0:-1.0;1.0:1.0:179.0]
+masses = 1.0:5.0:200
+n_frames = length(masses)
+anim = @animate for (i, mass) in enumerate(masses)
     print("Frame: $i / $n_frames complete")
-    fangle = Float64(angle)
-    temperature_render(obs_angle=fangle, tolerance = 1e-4)
+    # fangle = Float64(angle)
+    fmass = Float64(mass)
+    temperature_render(mass=mass, tolerance=1e-10, resolution=1080, size_multiplier=4, dtmax=2.5)
 end
-gif(anim, "hello.gif", fps=10)
\ No newline at end of file
+gif(anim, "massgif.gif", fps=20)
```

Comment:
Can you post the gif?

Reply:
![275733518_1007480233212804_3890997483161396099_n](https://user-images.githubusercontent.com/94124028/158189131-69379c85-4e87-457e-99b8-037d11c6fb16.gif)

This one looks pretty good, and we think everything here is physically correct. One small thing would be to centre the black hole on zero, so the size of the disc in gravitational radii (or some other unit?) could be a bit clearer.